regulations/static/regulations/js/source/views/main/star-processor.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+var $ = require('jquery');
+
+/***
+ * Functions to process the different types of "stars", indicators that text
+ * can be expanded. Each accepts an $li which may need to be collapsed in
+ * addition to an $expander to expand collapsed $li texts. Each method returns
+ * an updated $expander.
+ ***/
+var StarProcessor = {
```

Comment:
The name and casing of this variable sounds like a class to me, not like a bundle of utility functions. Could this be called something like `starHelpers`? Also, any reason for declaring this object of helpers with a name, then assigning to `module.exports`?